packages/dx-react-grid/src/plugins/table-column-reordering.tsx

```diff
@@ -75,33 +91,34 @@ class TableColumnReorderingRaw extends React.PureComponent {
     this.cellDimensions = [];
   }
 
-  ensureCellDimensionGetters(tableColumns) {
+  ensureCellDimensionGetters(tableColumns: TableColumn[]) {
```

Comment:
Why not `: ReadonlyArray<TableColumn>`? (it will guard against occasional changes that would attempt to modify `tableColumns`)